rosbag2_performance/rosbag2_performance_benchmarking/src/benchmark_publishers.cpp

```diff
@@ -55,44 +54,70 @@ class BenchmarkPublishers : public rclcpp::Node
   }
 
 private:
-  void create_benchmark_publishers_and_producers()
+  struct BenchmarkProducer
+  {
+    std::shared_ptr<msg_utils::ProducerBase> producer;
```

Comment:
Nitpick. Suggesting to rename `producer` to the `msg_producer` to avoid awkward `producer->producer` chain.

Reply:
updated